### DISCUSSION TITLE
Does LibreTube support PeerTube instances and if not, will it?

### DISCUSSION THREAD

**Discussion by tinkerpunktheprol** [libre-tube/LibreTube#3690] [Q&A] [answered]
I found a single post mentioning PeerTube, but the back and forth left me a bit confused. I would also like to be able to subscribe to one or more PeerTube instances alongside YouTube. So if it does support Peertube, is it possible to have multiple instances at once?

**Comment by Bnyro** [comment, marked as answer]
No, and not planned.
